src/simulate/README.md

```diff
@@ -0,0 +1,15 @@
+### Simulate sending caliper events to UDP
+ The Unizin Data Platform is intended to absorb the caliper events. UDP is development in progress and this application is taking caliper 1.1 events including envelope from flat files sends it to UDP
+
+### Running requirements
+1. The script is tested with Python 3.6.3
```

Comment:
This is a silly-ish comment:  Something that annoys me is useful programs being referred to as "scripts".  To me, scripts are small, simple pieces of code that do a useful task, but don't have much logic to them.  Something like the program you're writing here is more complex than that and deserves to be called a "program" or if it's big enough (I think it is), "app".